interface/orders/types.php

```diff
@@ -191,6 +191,8 @@ function refreshFamily(id, haskids) {
     event.preventDefault();
     console.log(event.target.href);
     $("#modal-iframe").iziModal({
+    title: 'Add procedure',
+    subtitle: 'Fill in procedure', headerColor: '#88A0B9',
```

Comment:
a break here will do .... headerColor is a key value

Reply:
Yes you are right. I have made the necessary changes to both new definitions.